docs/index.rst

```diff
@@ -142,10 +142,40 @@ sanity. To achieve this, the codebase is a collection of small reusable
 utilities and abstractions composed together in a few helpful layers.
 
 
+SessionManager
+--------------
+
+:doc:`SessionManager(...) <manager>` is currently urllib3's most abstract manager
```

Comment:
Great first take, thank you! I might edit some of this later, hope you don't mind. :)